WordPress/Classes/Users/Views/UserDetailsView.swift

```diff
@@ -61,9 +69,9 @@ struct UserDetailsView: View {
                 }
             }
 
-            if !applicationTokenListViewModel.applicationTokens.isEmpty {
-                Section(ApplicationTokenListView.title) {
-                    ForEach(applicationTokenListViewModel.applicationTokens) { token in
+            if !viewModel.applicationTokens.isEmpty {
+                Section(Strings.applicationPasswordSectionTitle) {
```

Comment:
I'd suggest showing "Application Passwords" as a separate screen and have only a navigation link form the user details. If the user has a bunch of these, you don't want to show them all here inline. In fact, it's not something I would expect users to access often.